src/System.Memory/ref/System.Memory.cs

```diff
@@ -91,13 +72,20 @@ public partial struct Span<T>
         public bool TryCopyTo(System.Span<T> destination) { throw null; }
     }
 
-    public static class SpanExtensions
+    public static class Span
```

Comment:
As this is effectively a breaking change (though only for the location of extension methods, which may not be a big deal), we should probably bump the assembly version to 4.1.0.0 in the ref and src projects.